### DISCUSSION TITLE
headphones icon

### DISCUSSION THREAD

**Discussion by vshelke** [tailwindlabs/heroicons#489] [Ideas]

**Comment by lxrst** [comment]
Looks like this got lost

**Comment by maiconsanson** [comment]
Recently, I have needed to use a headphones icon in a project that I already use the fucking awesome Herocions.
The main goal was to encourage the user to use the headphones for a better sound experience.
To my surprise, I didn't find any icon to fit that.

So here I am...

**Comment by ndri** [comment]
I added `headphones` and `headphones-slash` to [Sidekickicons](https://github.com/ndri/sidekickicons/), which are meant to be used as a companion to Heroicons. Browse them here: 
- https://sidekickicons.com/?icon=headphones
- https://sidekickicons.com/?icon=headphones-slash
